content/200-concepts/100-components/01-prisma-schema/06-relations/420-referential-integrity.mdx

```diff
@@ -35,37 +67,43 @@ generator js {
 }
 ```
 
-Setting the value to `prisma` will remove all foreign keys previously created in the next migration when using Prisma Migrate and the referential integrity will be handled by the client. The value `foreignKeys` creates foreign keys for all relations in the next migration when using Prisma Migrate, letting the database handle the referential integrity.
+To change the type of referential integrity supported, update the `referentialIntegrity` paramater in the [`datasource`](/reference/api-reference/prisma-schema-reference#datasource) block. The available options are:
+
+- `foreignKeys`: this handles referential integrity in the database using foreign keys. This is the default option when using all SQL database connectors.
+- `prisma`: this emulates referential integrity in the Prisma client. This is the default (and only available) option when using the MongoDB connector.
+
+<Admonition type="warning">
+  Setting the value to `prisma` will remove all previously created foreign keys
+  in the next migration when using Prisma Migrate. Setting the value to
+  `foreignKeys` will create foreign keys for all relations in the next migration
+  when using Prisma Migrate.
+</Admonition>
 
 ## Handling the referential integrity in Prisma
 
-If the integrity is handled by the Prisma Client, for now there will be reduced amount of available [referential actions](/concepts/components/prisma-schema/relations/referential-actions) available in the data model:
+If referential integrity is handled by the Prisma Client, there is currently a reduced number of supported [referential actions](/concepts/components/prisma-schema/relations/referential-actions) available in the data model:
 
-| action     | `onUpdate` | `onDelete` |
-|:-----------|:-----------|:-----------|
-| Restrict   | no         | yes        |
-| SetNull    | no         | yes        |
-| SetDefault | no         | no         |
-| Cascade    | no         | yes        |
-| NoAction   | yes        | yes        |
+| Referential action | `onUpdate` | `onDelete` |
+| :----------------- | :--------- | :--------- |
+| Restrict           | no         | yes        |
+| SetNull            | no         | yes        |
+| SetDefault         | no         | no         |
+| Cascade            | no         | yes        |
+| NoAction           | no         | yes        |
 
 In this mode, any [`raw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) database queries will not trigger any of the actions, and they should be handled manually if deleting or updating records using the [`raw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) queries.
```

Comment:
This feels kind of a limitation of the ref integrity mode `prisma`, so maybe we should call it out as that? Or a "design inhererent limitation" or sth

Reply:
Do you have a suggestion on how to phrase this?

Reply:
Either a headline "limitations" or just preface the paragraph with `Limitation: `.

Reply:
Done